modules/nginx/manifests/redirect.pp

```diff
@@ -0,0 +1,44 @@
+define nginx::redirect (
+    $web_domain		= hiera("web_domain"),
```

Comment:
We don't usually let non-oaeservice modules pull from hiera. I'd prefer it if the oaeservice/manifests/nginx.pp module would pass this information in.